test/bugs/1719.js

```diff
@@ -0,0 +1,52 @@
+// https://github.com/swagger-api/swagger-js/issues/1719
+
```

Comment:
Just following existing project examples. fyi, swagger-ui and swagger-editor do this too.

Reply:
Also, the `test/subtree-resolver` currently only runs a single test due to existence of a `test.only()`. There appears to be two breaking tests that are currently skipped.

Reply:
> Just following existing project examples. fyi, swagger-ui and swagger-editor do this too.

My questions are more: Why we're doing this? Do we know why this convention exists and what are the conditions to activate it? How do we benefit from it?

Reply:
> Also, the `test/subtree-resolver` currently only runs a single test due to existence of a `test.only()`. There appears to be two breaking tests that are currently skipped.

Damn...let's fix it and run all tests except the ones that don't pass ;]

Reply:
Creating a separate ticket for this topic, #1725

Reply:
As for this tests structure the contention that you've used here (`bugs` directory) currently only seems to apply to cases when the original bug was reported either on swagger-ui or swagger-editor repository as a public issue (let's call it remove issue). This doesn't seem to be the case for this PR. We've created an issue on swagger-js (#1719) and we've proceeded with solving it by `bugs` convention instead of incorporating the bugfix tests to existing related test suites.